bundles/io/org.openhab.io.caldav/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,53 @@
+Manifest-Version: 1.0
+Private-Package: org.openhab.io.caldav.internal
+Ignore-Package: org.openhab.io.caldav.internal
+Bundle-Name: openHAB CalDav Calendar
+Bundle-Vendor: openHAB.org
+Bundle-Version: 1.8.0.qualifier
+Bundle-Activator: org.openhab.io.caldav.internal.CalDavActivator
+Bundle-ManifestVersion: 2
+Bundle-License: http://www.eclipse.org/legal/epl-v10.html
+Bundle-Description: This is the CalDav enhancement of the open Home Aut
+ omation Bus (openHAB)
+Import-Package: com.google.common.collect;version="1.0.0",
```

Comment:
No, i can't. The library which i am using for loading and connecting to the caldav server requires this specific versions. Otherwise a newer one is taken and it's not working.